_includes/anchor_headings.html

```diff
@@ -92,6 +92,6 @@
         {{ include.bodySuffix }}
       </h{{ _workspace | last }}
     {% endcapture %}
-    {% capture edited_headings %}{{ edited_headings }}{{ new_heading | replace: '  ', '' | replace: new_line, '' }}{{ new_line }}{% endcapture %}
+    {% capture edited_headings %}{{ edited_headings }}{{ new_heading }}{{ new_line }}{% endcapture %}
```

Comment:
Since the code's no longer doing weird newline manipulation, there's no need for the `{{ new_line }}` at all; nor the `{% capture new_line %}` at the top.